packages/docs-page/server/loaders/file-system.ts

```diff
@@ -18,7 +20,9 @@ interface FileSystemLoaderOpts extends DataLoaderOpts {
   navDataFile: string
   localContentDir: string
   mainBranch?: string // = 'main',
-  remarkPlugins?: ((params?: ParsedUrlQuery) => Pluggable[]) | Pluggable[]
+  remarkPlugins?:
+    | ((params?: ParsedUrlQuery, version?: string) => Pluggable[])
```

Comment:
@BRKalow What do you think about making this callback func send a `ctx` object, with fields like `ctx.params` & `ctx.version`...

(Would be a Major/breaking change)

Reply:
Hmm, my gut says no for now as we're just creating more work for ourselves to update any existing usages. 💭